pkg/cmdutil/json_flags.go

```diff
@@ -121,6 +173,18 @@ type exportFormat struct {
 	template string
 }
 
+// NewJSONExporter returns an Exporter to emit JSON.
+func NewJSONExporter(opts ...JSONExporterOption) Exporter {
```

Comment:
It needs to have access to the `exportFormat` struct, though - or should I just duplicate it? I was hoping to avoid the latter. Or do I make `exportFormat` public instead? That's really all that's needed - just to return an implementation of `cmdutil.Exporter`.

Reply:
I removed the options pattern, but kept the public function. And while I realize there's only one format supported now, I know more have been considered e.g., CSV. As such, I renamed the generic `exportFormat` to `jsonExporter`. I also considered just making the whole thing public, but perhaps a factory function would be better if ever needed - to switch on the format. Seemed overkill right now, though I'm happy to implement other format sometime.

Feel free to resolve if this is fine, or give me some other idea of what you'd like best.